### PR TITLE
Fix update SignalR bugs and add tests

### PR DESCRIPTION
**Pull request by Y-Sindo**

### CODE REVIEW

File: src/SignalR/SignalR/Az.SignalR.psd1 [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.4.0'
```
Comment (msJinLei) [inline src/SignalR/SignalR/Az.SignalR.psd1:15]: Please revert the change of the line. We will update the version using automatic tools

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/SignalR/SignalR/Az.SignalR.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
+ModuleVersion = '1.4.1'
```
Comment (Y-Sindo) [inline src/SignalR/SignalR/Az.SignalR.psd1:15]:
```suggestion
ModuleVersion = '1.4.0'
```

**Review by msJinLei — APPROVED** [review]
